frontend/src/metabase/visualizations/components/ObjectDetail/ObjectDetailView.unit.spec.tsx

```diff
@@ -468,9 +468,9 @@ describe("ObjectDetailView", () => {
     expect(
       screen.queryByText("Choose a record to update"),
     ).not.toBeInTheDocument();
-    expect(screen.getByText("Loading...")).toBeInTheDocument();
+    expect(screen.getByTestId("loading-spinner")).toBeInTheDocument();
```

Comment:
here we test that loading exists and then disappears from the dom. and it's the logic of `waitForElementToBeRemove` actually. Do you think we should check both in the util? (that element first appears then disappears)

Reply:
> Do you think we should check both in the util?

The util was created to avoid exactly that.

True, if both assertions are needed (there aren't that many places like that though), then `waitForElementToBeRemoved` would be the right choice, although I prefer explicit code, like here.

----

If we ever get async issues with this assertion, we could add another util, like this:
```tsx
await waitForLoadingSpinnerToApppear();
await waitForLoadingSpinnerToDisappear();
```

Reply:
no objections